Makefile.am

```diff
@@ -132,11 +132,11 @@ install: src/dosbox-x
 	install -m 644 $(srcdir)/contrib/linux/dosbox-x.1 $(DESTDIR)$(mandir)/man1
 	mkdir -p $(DESTDIR)$(prefix)/share/bash-completion/completions
 	install -m 644 $(srcdir)/contrib/linux/dosbox-x $(DESTDIR)$(prefix)/share/bash-completion/completions
-	-test -x /usr/sbin/setcap && setcap cap_net_raw=ep $(DESTDIR)$(bindir)/dosbox-x
+	-command -v setcap >/dev/null && setcap cap_net_raw=ep $(DESTDIR)$(bindir)/dosbox-x || true

```

Comment:
What is this supposed to accomplish? I don't fully understand the point of `command` here.

And piping to `true` at the end, why? I understand to always return 0 as return code, but again why? The `-` at the start of the line allows a non-zero return code without halting the execution, so why the hack?

Reply:
Instead of command, the following would also work to check if setcap is available as an executable.
```
test -x $(which setcap)
```
That would work from a batch script. Not tested from a makefile.

Reply:
I'll get back to you after doing some testing but I'm pretty sure that make will fail the build if it can't find the command to run, the dash just spews an error but continues it if the command returns a negative value. The || true here is to just make sure command's negative value isn't taken as a failure.

You could use text -x $(which setcap) but there's numerous online horror stories of programs that expect which to be installed when it's not so I just wanted to avoid it. Instead using command will check if setcap exists.

Reply:
regarding `|| true`, I don't see the harm in letting it print an error. Due to the line starting with a dash, the error will not be fatal to the execution anyway. In fact, I think letting it print an error if it cannot execute setcap if a positive thing as you know there may be an issue.